internal/ui/components/home.templ

```diff
@@ -3,8 +3,98 @@ package components
 templ Home() {
 	@Base() {
 		<h1>PHIN VADS Home</h1>
-		<div class="vh">
-			50 vh content foobar
+		<div class="search">
+			<p>Search PHIN VADS</p>
+			<div class="grid-row">
+				<form class="usa-form">
+					<section class="grid-col-auto">
+					<select class="usa-select" name="options" id="options">
+						<option value> All Vocabulary </option>
+						<option value="value1">Option A</option>
+						<option value="value2">Option B</option>
+						<option value="value3">Option C</option>
+					</select>
+					</section>
+				</form>
+				<section class="grid-col-fill" aria-label="Big search component">
+					<form class="usa-search usa-search--big" role="search">
+						<label class="usa-sr-only" for="search-field-en-big">Search</label>
+							<input
+							class="usa-input"
+							id="search-field-en-big"
+							type="search"
+							name="search"
+							/>
+						<button class="usa-button" type="submit">
+						<span class="usa-search__submit-text">Search </span>
+						<img
+							src="/assets/img/usa-icons-bg/search--white.svg"
+							class="usa-search__submit-icon"
+							alt="Search"
+						/>
+						</button>
+					</form>
+				</section>
+			</div>
+			<div class="grid-row">
+				<div class="grid-col-8 grid-offset-8">
```

Comment:
can get rid of this div, and just have the `<p>` nested directly under the `grid-row` div; if you then add a style to that remaining div (maybe add a unique class name), and set `justify-contents: flex-end`, the Advanced Search button will stay all the way over to the right! (Might need to throw some extra margin px in there to make it line up perfectly)

Reply:
took some tweaking with the other search fields, but I got it fixed eventually in a way that didn't go wonky if you get down to smaller window sizes.